browser/extensions/api/settings_private/brave_prefs_util.cc

```diff
@@ -14,6 +14,7 @@
 #include "brave/components/brave_wallet/browser/pref_names.h"
 #include "brave/components/brave_wayback_machine/buildflags.h"
 #include "brave/components/crypto_dot_com/browser/buildflags/buildflags.h"
+#include "brave/components/de_amp/pref_names.h"
```

Comment:
nothing should be in the roof of the component if you're using browser, common, etc...

Reply:
and pref_names should go in common